lib/arborist/rebuild.js

```diff
@@ -22,7 +22,7 @@ const _linkAllBins = Symbol('linkAllBins')
 const _runScripts = Symbol('runScripts')
 const _buildQueues = Symbol('buildQueues')
 const _addToBuildSet = Symbol('addToBuildSet')
-const _checkBins = Symbol('checkBins')
+const _checkBins = Symbol.for('checkBins')
```

Comment:
why use Symbol.for instead of exporting a symbol that can be used explicitly?

Reply:
No compelling reason, really.  We use `Symbol.for()` in a few other places as well, when logic has to be shared between mixins.  Would be fine for a later refactor.